File: tops-work/SE/Assignment/Module-2(Fundamentals-of-Programming)/String/que13.c
/*
Que13. Write a program in C to remove characters from a string except alphabets.
*/

#include <stdio.h>
#include <string.h>

int main()
{
    char str[50];
    printf("Enter any string\n");
    scanf("%s",str);
    int len = strlen(str);
    for(int i = 0; i < len; i++)
    {
        if((str[i] >= 'a' && str[i] <= 'z') || (str[i] >= 'A' && str[i] <= 'Z'))
        {
            printf("%c",str[i]);
        }
        else
        {
            continue;
        }
    }
    
    return 0;
}

File: tops-work/PYTHON/Assignment/Fundamentals-of-Python/que13.py
"""
Que13. Write a Python program to count the number of characters (character
frequency) in a string 
"""

name = input("Enter any string: ").lower()
empty_str = ''
for ch in name:
    if ch not in empty_str:
        empty_str += ch
        print(f"{ch} - {str(name.count(ch))}")